core/src/main/java/org/n52/iceland/ogc/filter/TemporalFilter.java

```diff
@@ -0,0 +1,105 @@
+/*
+ * Copyright 2015 52°North Initiative for Geospatial Open Source
+ * Software GmbH
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.n52.iceland.ogc.filter;
+
+import org.n52.iceland.ogc.filter.FilterConstants.TimeOperator;
+import org.n52.iceland.ogc.gml.time.Time;
+
+/**
+ * Temporal filter class
+ *
+ * @since 1.0.0
+ *
+ */
+public class TemporalFilter extends Filter<TimeOperator> {
+
+    private TimeOperator operator;
+
+    private Time time;
+
+    public TemporalFilter() {
+        this((String)null, null, null);
```

Comment:
Better use the `TimeOperator` constructor... This one will throw a `IllegalArgumentException`.